File: 1015/1015.c
#include <stdio.h>
#include <math.h>
double dis(double a, double b, double c, double d);
int main() {
    double x1,x2,y1,y2,s;
    scanf("%lf %lf",&x1,&y1);
    scanf("%lf %lf",&x2,&y2);
    s=dis(x1,x2,y1,y2);
    printf("%.4lf\n",s);
    return 0;
}

double dis(double a, double b, double c, double d)
{
    return sqrt(pow((b-a),2)+pow((d-c),2));
}

File: Python/1015.py
import math
def dis(a,b,c,d):
    return float(math.sqrt(pow((b-a),2)+pow((d-c),2)))

x,y=([] for i in range(2))
for i in range(2):
    xa,ya=input().split()
    x.append(float(xa))
    y.append(float(ya))

s=dis(x[0],x[1],y[0],y[1])
print(f"{s:.4f}")